scripts/automator/packages/manager-apt

```diff
@@ -0,0 +1,3 @@
+# Package repo: https://packages.ubuntu.com/
+# No differentiation between 32-bit and 64-bit
```

Comment:
Will drop. Hopefully we never need 32bit under *nix anymore.

Yes, 32 bit wonderful under msys2, because the binary names are identical, so is basically a  no-op.

Reply:
Fixed in 46b9906a